From a1c191a020f6eebc9a94b641be1fca7885fc9dd2 Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@ti.com>
Date: Mon, 9 Sep 2019 17:05:48 -0400
Subject: [PATCH 1/2] km: support OpenEmbedded hardfp toolchain w/o -gnueabihf
 suffix

OE toolchains are not having armhf in its name but they still are armhf
ABI, teach this to sgx module build system

recognize musleabi in tuples as well

Add yoe distro support

Upstream-Status: Pending

Signed-off-by: Denys Dmytriyenko <denys@ti.com>
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 eurasia_km/eurasiacon/build/linux2/config/compiler.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/eurasia_km/eurasiacon/build/linux2/config/compiler.mk b/eurasia_km/eurasiacon/build/linux2/config/compiler.mk
index dead2f9..1b5f090 100644
--- a/eurasia_km/eurasiacon/build/linux2/config/compiler.mk
+++ b/eurasia_km/eurasiacon/build/linux2/config/compiler.mk
@@ -68,12 +68,12 @@ define calculate-compiler-preferred-target
   ifneq ($$(filter aarch64-%,$$($(1)_compiler_preferred_target)),)
    $(1)_compiler_preferred_target := aarch64-linux-gnu
   endif
+  ifneq ($$(filter arm-%-gnueabihf arm-oe-linux-gnueabi arm-poky-linux-gnueabi arm-yoe-linux-gnueabi arm-oe-linux-musleabi arm-poky-linux-musleabi arm-yoe-linux-musleabi,$$($(1)_compiler_preferred_target)),)
+   $(1)_compiler_preferred_target := arm-linux-gnueabihf
+  endif
   ifneq ($$(filter arm-%-gnueabi armv7a-cros-linux-gnueabi armv7hl-redhat-linux-gnueabi,$$($(1)_compiler_preferred_target)),)
    $(1)_compiler_preferred_target := arm-linux-gnueabi
   endif
-  ifneq ($$(filter arm-%-gnueabihf,$$($(1)_compiler_preferred_target)),)
-   $(1)_compiler_preferred_target := arm-linux-gnueabihf
-  endif
  endif
 endef
 
-- 
2.34.1

